resources/definitions/deltacomb.def.json

```diff
@@ -55,6 +56,7 @@
         "support_use_towers" :                   { "default_value": false },
         "jerk_wall_0" :                          { "value": "30" },
         "jerk_travel" :                          { "default_value": 20 },
-        "acceleration_travel" :                  { "value": 10000 }
+        "acceleration_travel" :                  { "value": 10000 },
+        "machine_max_feedrate_z" :               { "default_value": 150 }
```

Comment:
Is this setting change also needed? At least this is not part of the diameter fix.

Reply:
Yes It's needed. I forgot it in a previous commit. will force the Z speed to 150 because in delta machines can handle it better